src/app.js

```diff
@@ -1 +1,24 @@
+/* eslint-disable no-console */
 'use strict';
+
+const fs = require('fs');
+
+function copyFile() {
+  const [src, dest] = process.argv.slice(2);
+
+  if (!dest) {
+    console.error('Please provide two arguments');
+  }
+
+  try {
+    if (src && dest && src !== dest) {
```

Comment:
The condition `src !== dest` is checked, which is good to prevent copying a file onto itself. However, ensure that this logic is intended and correctly handles cases where the source and destination are the same.